Nodejs/Product/Nodejs/Snippets/1033/Nodejs/exports.snippet

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>export</Title>
+      <Description>Code Snippet for exporting a module</Description>
+      <Shortcut>export</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Object>
+          <ID>myModule</ID>
+          <Default>myModule</Default>
+        </Object>
+        <Object>
+          <ID>myVar</ID>
+          <Default>myVar</Default>
+        </Object>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[module.exports.$myModule$ = $myVar$;]]>

```

Comment:
I think this snippet is a bit too specific of a scenario - people assign all sorts of things to module.exports (not just variables, so we would really just want to end at the equals so people can start typing whatever they want), and I don't think we're saving them much time here with ```module.exports.$myModule$ = ```, so let's just remove this snippet.

(take a look at the following query to see how people use module.exports in real code)
https://github.com/search?p=1&q=module.exports&ref=searchresults&type=Code&utf8=%E2%9C%93